Description: fix haddock in Ccache.hs
 The '^' breaks the build of libghc-propellor-doc
Author: Sean Whitton <spwhitton@spwhitton.name>
Forwarded: yes

--- a/src/Propellor/Property/Ccache.hs
+++ b/src/Propellor/Property/Ccache.hs
@@ -100,7 +100,7 @@ group@(Group g) `hasCache` limit = (make
 					, groupWriteMode
 					, setGroupIDMode
 					]) `onChange` fixSetgidBit
-				-- ^ we use onChange to catch upgrades from
+				-- here, we use onChange to catch upgrades from
 				-- 3.0.5 where the setGroupIDMode line was not
 				-- present
 			& hasLimits path limit
